.gitignore

```diff
@@ -1,7 +1,7 @@
 .idea/
 .vscode/
 *.iml
-.env
+cinema_service/.env
```

Comment:
It must stored inside root folder instead of config folder

Reply:
do it